Kickstarter-iOS/Views/Controllers/FindFriendsViewController.swift

```diff
@@ -33,9 +34,32 @@ internal final class FindFriendsViewController: UITableViewController {
     self.navigationController?.setNavigationBarHidden(false, animated: animated)
   }
 
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableControllerStyle()
+      |> UIViewController.lens.title %~ { _ in Strings.Follow_friends() }
+
+    _ = self.navigationController?.navigationBar
+      ?|> baseNavigationBarStyle
+
+    _ = self.loadingIndicatorView
+      |> UIActivityIndicatorView.lens.activityIndicatorViewStyle .~ .white
+      |> UIActivityIndicatorView.lens.color .~ .ksr_navy_900
+  }
+
   override func bindViewModel() {
     super.bindViewModel()
 
+    self.loadingIndicatorView.rac.animating = self.viewModel.outputs.loaderIsAnimating
+
+    self.viewModel.outputs.loaderIsAnimating
+      .observeForUI()
+      .observeValues { [weak self] isAnimating in
+        self?.tableView.tableFooterView = isAnimating ? self?.loadingIndicatorView : nil
+    }
```

Comment:
we talked about this in squash, but we should keep our loader positions consistent throughout the app. I can help if this wasn't centering properly.